cmd/wharf/provisioner.go

```diff
@@ -4,14 +4,10 @@ import (
 	"github.com/iver-wharf/wharf-cmd/pkg/provisioner"
 	"github.com/spf13/cobra"
 	"k8s.io/client-go/rest"
-	"k8s.io/client-go/tools/clientcmd"
 )
 
 var provisionerFlags = struct {
-	k8sOverrides clientcmd.ConfigOverrides
-
 	restConfig *rest.Config
-	namespace  string
```

Comment:
These `restConfig` and `namespace` fields in here has always bugged me as they don't really belong in this struct.

Very nice you got rid of `namespace`, but is it possible to rid `restConfig` as well? Could rely on only the return value from `loadKubeconfig` right? Don't need this global